test/clj/rems/test_browser.clj

```diff
@@ -985,7 +984,6 @@
       (click-navigation-menu "Catalogue")
       (click-navigation-menu "Actions")
       (btu/scroll-and-click [:handled-applications-collapse {:fn/text "Show more"}])
-      (btu/wait-for-animation)
```

Comment:
Now this doesn't wait before taking the screenshot?

Reply:
Waiting is now done inside `btu/scroll-and-click`, by `rems.browser-test-util/wait-for-idle` which is otherwise similar but requests browser to return when it has idle time.

Reply:
Hmm, actually that sounds wrong, `scroll-and-click` uses wait initially before the click. But I think waiting should happen in `screenshot` function also? Since `wait-for-idle` uses idle callback, it might return immediately if browser is ready.

Reply:
One way or another this spot needs a little bit of waiting before taking the screenshot or otherwise it'll show incomplete state.